index.html

```diff
@@ -12340,29 +12383,6 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 					</tr>
 				</tbody>
 			</table>
-			<table class="value-descriptions">
-				<caption>Values:</caption>
```

Comment:
Curious how the other editors feel about removing the values table.

Reply:
looks like this table was not removed, just moved up and expanded?

Reply:
I suspect that for some attributes we might end up removing it if they are a simpler type. We only need a table for enumerated attributes when defining them in terms of HTML attribute types.